packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -205,25 +213,40 @@ export class IndexedDbPersistence implements Persistence {
 
   private queryCache: IndexedDbQueryCache;
   private remoteDocumentCache: IndexedDbRemoteDocumentCache;
+  private webStorage?: Storage;
+  private listenSequence: ListenSequence;
 
   constructor(
     private readonly persistenceKey: string,
     private readonly clientId: ClientId,
     platform: Platform,
     private readonly queue: AsyncQueue,
     serializer: JsonProtoSerializer,
-    synchronizeTabs: boolean
+    private readonly multiClientParams?: {
+      sequenceNumberSyncer: SequenceNumberSyncer;
+    }
```

Comment:
Suggestion: Can we always pass this and have a no-op implementation?

Reply:
I'm not wild about that, especially if the `synchronizeTabs` flag is still present. It lets callers specify invalid configurations, mismatching the flag and the params.